test/fail_compilation/fail15068.d

```diff
@@ -0,0 +1,16 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/fail15068.d(15): Error: template instance `T!int` `T` is not a template declaration, it is a type
```

Comment:
Can we change the error message to something along the lines of:

```
Error: `T!int` is not a valid template instance, because `T` is not a template declaration, but a type (`` == int`).
```

(i.e. is the `template instance` part after `Error: ` mandatory, can we override it?)

Reply:
Nice idea. Done.